src/locales/id.json

```diff
@@ -0,0 +1,95 @@
+{
+  "labels": {
+    "paste": "Tempel",
+    "selectAll": "Pilih Semua",
+    "copy": "Salin",
+    "bringForward": "Memajukan",
+    "sendToBack": "Bawa ke Belakang",
+    "bringToFront": "Bawa ke Depan",
+    "sendBackward": "Kirim Mundur",
+    "delete": "Hapus",
+    "copyStyles": "Salin Style",
+    "pasteStyles": "Tempel Style",
+    "stroke": "Stroke",
+    "background": "Latar",
+    "fill": "Fill",
+    "strokeWidth": "Lebar Stroke",
+    "sloppiness": "Sloppiness",
+    "opacity": "Opacity",
+    "fontSize": "Ukuran Font",
+    "fontFamily": "Font Family",
+    "onlySelected": "Hanya yang Dipilih",
+    "withBackground": "Dengan Latar",
+    "handDrawn": "Hand-Drawn",
+    "normal": "Normal",
+    "code": "Kode",
+    "small": "Kecil",
+    "medium": "Medium",
+    "large": "Besar",
+    "veryLarge": "Sangat Besar",
+    "solid": "Solid",
+    "hachure": "Hachure",
+    "crossHatch": "Cross-Hatch",
+    "thin": "Lembut",
+    "bold": "Tebal",
+    "extraBold": "Sangat Tebal",
+    "architect": "Arsitek",
+    "artist": "Artis",
+    "cartoonist": "Cartoonist",
+    "fileTitle": "Judul Berkas",
+    "colorPicker": "Pilihan Warna",
+    "canvasBackground": "Latar Kanvas",
+    "drawingCanvas": "Drawing Canvas",
+    "layers": "Lapisan",
+    "language": "Bahasa"
+  },
+  "buttons": {
+    "clearReset": "Setel Ulang Kanvas",
+    "export": "Ekspor",
+    "exportToPng": "Expor ke PNG",
+    "exportToSvg": "Expor ke SVG",
```

Comment:
I don’t speak Indonesian but I wanted to make sure this wasn’t a typo (“Ekspor” above, “Expor” here).

Reply:
Fixed in next commit